README.md

```diff
@@ -18,128 +18,177 @@ your part.
+### What's the justification that this really does need to be signed for the whole world to be able to boot it?
 -------------------------------------------------------------------------------
 [your text here]
 
 -------------------------------------------------------------------------------
-Who is the primary contact for security updates, etc.
+### Who is the primary contact for security updates, etc.?
 -------------------------------------------------------------------------------
 - Name:
 - Position:
 - Email address:
 - PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the Linux community:
 
 -------------------------------------------------------------------------------
-Who is the secondary contact for security updates, etc.
+### Who is the secondary contact for security updates, etc.?
 -------------------------------------------------------------------------------
 - Name:
 - Position:
 - Email address:
 - PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the Linux community:
 
 -------------------------------------------------------------------------------
-Please create your shim binaries starting with the 15.4 shim release tar file:
-https://github.com/rhboot/shim/releases/download/15.4/shim-15.4.tar.bz2
+### What changes were made since your SHIM was last signed?
+-------------------------------------------------------------------------------
+`[your text here]`
+
+-------------------------------------------------------------------------------
+### Were these binaries created from the 15.4 shim release tar?
+Please create your shim binaries starting with the 15.4 shim release tar file: https://github.com/rhboot/shim/releases/download/15.4/shim-15.4.tar.bz2
+
+This matches https://github.com/rhboot/shim/releases/tag/15.4 and contains the appropriate gnu-efi source.
 
-This matches https://github.com/rhboot/shim/releases/tag/15.4 and contains
-the appropriate gnu-efi source.
 -------------------------------------------------------------------------------
-[Please confirm]
+[your text here]
 
 -------------------------------------------------------------------------------
-URL for a repo that contains the exact code which was built to get this binary:
+### URL for a repo that contains the exact code which was built to get this binary:
 -------------------------------------------------------------------------------
 [your url here]
 
 -------------------------------------------------------------------------------
-What patches are being applied and why:
+### What patches are being applied and why:
 -------------------------------------------------------------------------------
 [your text here]
 
 -------------------------------------------------------------------------------
-If bootloader, shim loading is, GRUB2: is CVE-2020-14372, CVE-2020-25632,
```

Comment:
This, combined with the next two questions, seems like a multi-part "If you're loading grub2". This is now moved to the readme [here](https://github.com/rhboot/shim-review/pull/207/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R128-R145). Does it seem like I've gotten the meaning right?

Reply:
I think so, yes.